### DISCUSSION TITLE
How to create hidden page but include in sitemap

### DISCUSSION THREAD

**Discussion by paulocoutinhox** [getpelican/pelican#3118] [Q&A] [answered]
Hi,

When i create a page, an automatic link is created on menu for that page.

I remove it from menu using status=hidden.

But when i set status=hidden the page is removed from sitemap too.

How can i prevent it to be removed from sitemap?

Thanks.

**Comment by Tseing** [comment, marked as answer]
Hi @paulocoutinhox,

It seems that you want to enumerate menu items by yourself or do not want to display pages on menu. I suggest to modify setting in `pelicanconf.py`.

```py
# Do not display pages on menu but in sitemap
DISPLAY_PAGES_ON_MENU = False
# Do not display categories page on menu but in sitemap
DISPLAY_CATEGORIES_ON_MENU = False
# Pages, articles or whatever you want to display on menu
MENUITEMS = [("Home", SITEURL + "/index.html"),
             ("Categorias", SITEURL + "/categories.html"),
             ("Tools", SITEURL + "/pages/tools.html"),
             ("Search", "https://bing.com"),
             ("About", SITEURL + "/pages/about.html")]
```

**Reply by paulocoutinhox** [reply]
It this that i do now, i disabled all pages on menu. Thanks for this.
